assets/js/frontend/components/order-by-select/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+
+const OrderBySelect = ( { componentId, onChange, readOnly, value } ) => {
+	const selectId = `wc-block-reviews-by-product__orderby__select-${ componentId }`;
```

Comment:
Not generic enough classname. This component is specific to reviews.

If we want shared components for reviews, we should name them as such. e.g. `review-order-select`

Reply:
Right. Should be fixed in fde9e972de5bba44d64fe736d72f15705bb0de47.